lib/initPTB.m

```diff
@@ -121,15 +123,20 @@
 
         
     else
-        
+                
         audio_dev       = PsychPortAudio('GetDevices');
-        idx             = find([audio_dev.NrInputChannels] == 0 & [audio_dev.NrOutputChannels] == 2);
+        idx             = find([audio_dev.NrInputChannels] == 0 & ...
+                               [audio_dev.NrOutputChannels] == 2 & ...
+                               ~cellfun(@isempty, regexp({audio_dev.HostAudioAPIName},'WASAPI')));
         cfg.audio       = [];
         cfg.audio.i     = audio_dev(idx).DeviceIndex;
         cfg.fs          = audio_dev(idx).DefaultSampleRate;
         cfg.audio.channels = audio_dev.NrOutputChannels;
-        %cfg.audio.channels = 1; % we have mono sound actually
-        cfg.pahandle    = PsychPortAudio('Open',cfg.audio.i,1,1,cfg.fs,cfg.audio.channels);
```

Comment:
I would not delete this yet. Not sure which driver it was using by default, but maybe with WASAPI it will work with low latency mode.